src/ddmd/dcast.d

```diff
@@ -2666,28 +2666,6 @@ extern (C++) bool isVoidArrayLiteral(Expression e, Type other)
     return (e.op == TOKarrayliteral && t.ty == Tarray && t.nextOf().ty == Tvoid && (cast(ArrayLiteralExp)e).elements.dim == 0);
 }
 
-// used by deduceType()
```

Comment:
Seems like a useful comment.  I don't recommend deleting it.

Reply:
it is right above the deduceType helper which is right above deduceType. A rgrep after rawTypeMerge will show that deduceType is the only function using it. The only way to get to that function is by reading the deduceType function. It made sense to have that comment when it was in a different file, but now it seems kind of useless.

Reply:
Ok, I get it.  Thanks.